### PR TITLE
feat: second flywheel

### PR DESCRIPTION
**Pull request by ProgrammingSR**
made code to support the second flywheel

### CODE REVIEW

**Review by ArnoUUU — APPROVED** [review]

File: src/main/java/frc/robot/subsystems/ShooterSubsystem.java [hunk 1]
```diff
@@ -68,32 +72,56 @@ public ShooterSubsystem() {
...
+			SmartDashboard.putNumber("Shooter PID position error", m_blackShooterPID.getPositionError());
+			SmartDashboard.putNumber("Shooter PID position error", m_greenShooterPID.getPositionError());
+			SmartDashboard.putNumber("Shooter Feedforward output",
+					m_feedforward.calculate(m_blackShooterPID.getSetpoint()));
+			SmartDashboard.putNumber("Shooter Feedforward output",
+					m_feedforward.calculate(m_greenShooterPID.getSetpoint()));
+			SmartDashboard.putNumber("Shooter Power", m_blackFlywheel.get());
+			SmartDashboard.putNumber("Shooter Power", m_greenFlywheel.get());
+			SmartDashboard.putNumber("Shooter RPM", Utils.toRPM(m_blackFlywheel.getSelectedSensorVelocity()));
+			SmartDashboard.putNumber("Shooter RPM", Utils.toRPM(m_greenFlywheel.getSelectedSensorVelocity()));
```
Comment (LennyAbbas) [inline src/main/java/frc/robot/subsystems/ShooterSubsystem.java:124]: There are multiple SmartDashboard calls that have the same name when they do different things. This means the one that is called later will always overwrite the one that is called first. (ie lines 113 and 114 both print under "Shooter PID velocity error" when one should be "Shooter PID velocity error Green" and the other should be "Shooter PID velocity error Black"

**Review by LennyAbbas — CHANGES_REQUESTED** [review]
Overall this was done very cleanly and I have no complaints about the formatting. I'm noticing that there are some optimizations that should be made but not on this PR. Only problem is with the SmartDashboard calls.

File: src/main/java/frc/robot/subsystems/ShooterSubsystem.java [hunk 2]
```diff
@@ -156,8 +187,20 @@ public void intakeOff() {
...
+			m_sideFeeders.set(0);
+		} else {
+			m_sideFeeders.set(ShooterConstants.kSideFeederSpeed);
+		}
+		if (OIConstants.kTelemetry) {
+			SmartDashboard.putNumber("Target Shooter Speed", RPM);
+		}
+	}
+
+	public void setGreenShooterSpeed(double RPM) {
```
Comment (LennyAbbas) [inline src/main/java/frc/robot/subsystems/ShooterSubsystem.java:202]: This method needs docs and `setBlackShooterSpeed` also needs updated docs

File: src/main/java/frc/robot/subsystems/ShooterSubsystem.java [hunk 3]
```diff
@@ -34,14 +34,16 @@
...
+	private final WPI_TalonFX m_blackFlywheel = new WPI_TalonFX(ShooterConstants.kBlackFlywheelPort);
+	private final WPI_TalonFX m_greenFlywheel = new WPI_TalonFX(ShooterConstants.kGreenFlywheelPort);
 
 	private final MUX m_MUX = new MUX();
 	private final REVColorSensorV3 m_proximitySensor = new REVColorSensorV3(m_MUX, Port.kTwo);
 
 	// TODO tune
-	private final PIDController m_armPID = new PIDController(0.005, 0, 0);
-	private final PIDController m_shooterPID = new PIDController(0.0005, 0, 0);
+	private final PIDController m_armPID = new PIDController(ShooterConstants.kArmP, 0, 0);
```
Comment (LennyAbbas) [inline src/main/java/frc/robot/subsystems/ShooterSubsystem.java:44]: Creating a constant for ArmP is not relevant to this PR but it can stay.

**Pull request merged**